templates/kubernetes/data-sources/vkcs_kubernetes_clustertemplate.md.tmpl

```diff
@@ -12,6 +12,5 @@ description: |-
 
 ## Example Usage
 
-{{tffile .ExampleFile}}
-{{tffile "templates/kubernetes/data-sources/vkcs_kubernetes_clustertemplate/with_version/main.tf"}}
+{{tffile "examples/kubernetes/clustertemplate/main.tf"}}
```

Comment:
why do not .ExampleFile?

Reply:
it will not work for datasources i asked

Reply:
@schirevko could you comment this?